test/integration/operator_test.go

```diff
@@ -210,18 +166,66 @@ func checkResourcesAfterDelete(t testing.TB, namespace string, testName Test) {
 	}
 }
 
-func newKAC(namespace string, resources map[string]any) *unstructured.Unstructured {
-	return &unstructured.Unstructured{
-		Object: map[string]interface{}{
-			"kind": "KubeArchiveConfig",
-			"apiVersion": fmt.Sprintf("%s/%s",
-				kubearchivev1alpha1.SchemeBuilder.GroupVersion.Group,
-				kubearchivev1alpha1.SchemeBuilder.GroupVersion.Version),
-			"metadata": map[string]string{
-				"name":      kacName,
-				"namespace": namespace,
+// Test that having both a global and local KubeArchiveConfig works. This is done by archiving Pods
+// in the global KubeArchiveConfig and Jobs in the local KubeArchiveConfig. Then run a job and verify
+// that the job logs can be retrieved.
+func TestGlobalAndLocalKAC(t *testing.T) {
+	t.Helper()
+
+	globalres := map[string]any{
+		"resources": []map[string]any{
+			{
+				"selector": map[string]string{
+					"apiVersion": "v1",
+					"kind":       "Pod",
+				},
+				"archiveWhen":  "status.phase == 'Succeeded'",
+			},
+		},
+	}
+	localres := map[string]any{
```

Comment:
My local tests indicate that if the localres is empty, the local namespace is not configured for archival because it is not included in the `sink-filters` so the sink does not know that it should archive anything there, as if was not enabled in the first place. So add a test to confirm this, so with an empty `localres`.

Reply:
Good catch. I think you're right. I'll check this as soon as I get back.

Reply:
This will be handled in https://github.com/kubearchive/kubearchive/issues/948.